internal/cmd/root.go

```diff
@@ -150,6 +149,17 @@ func Execute() {
 	}
 }
 
+func setupAppWithProgressBar(cmd *cobra.Command) (*app.App, error) {
+	program := os.Getenv("TERM_PROGRAM")
+
+	if program == "ghostty" {
+		fmt.Fprintf(os.Stderr, ansi.SetIndeterminateProgressBar)
+		defer fmt.Fprintf(os.Stderr, ansi.ResetProgressBar)
+	}
+
+	return setupApp(cmd)
+}
```

Comment:
If we decide to proceed, we might need to think a little bit more about this.

This works for Ghostty, but looks like we wouldn't be able to detect the Windows Terminal from outside of a Bubble Tea instance? 🤔 

/cc @aymanbagabas